.eslintrc.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-    extends: ['@tybalt/ts-recommended'],
-    parser: '@typescript-eslint/parser',
+    extends: ['plugin:@tybalt/eslint-plugin/ts-recommended'],
```

Comment:
This is the suggestion from the docs: https://eslint.org/docs/latest/use/configure/configuration-files#using-a-configuration-from-a-plugin